Userland/Libraries/LibJS/Runtime/TypedArrayPrototype.cpp

```diff
@@ -1512,6 +1513,39 @@ JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::to_locale_string)
     return js_string(vm, builder.to_string());
 }
 
+// 1.2.2.1.3 %TypedArray%.prototype.toReversed
+JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::to_reversed)
+{
+    // 1. Let O be the this value.
+    // 2. Perform ? ValidateTypedArray(O).
+    auto* typed_array = TRY(validate_typed_array_from_this(global_object));
+
+    // 3. Let length be O.[[ArrayLength]].
+    auto length = typed_array->array_length();
+
+    MarkedVector<Value> arguments(vm.heap());
+    arguments.empend(length);
+    // 4. Let A be ? TypedArrayCreateSameType(O, « 𝔽(length) »).
+    auto* return_array = TRY(typed_array_create_same_type(global_object, *typed_array, move(arguments)));
+
+    // 5. Let k be 0.
+    // 6. Repeat, while k < length
+    for(size_t k = 0; k < length; k++){
+          // a. Let from be ! ToString(𝔽(length - k - 1)).
+          // Let Pk be ! ToString(𝔽(k)).
+          auto from = length - k - 1;
+
+          // c. Let fromValue be ! Get(O, from).
+          auto from_value = MUST(typed_array->get(from));
+
+          // d. Perform ! Set(A, Pk, kValue, true).
+          MUST(return_array->set(k, from_value, Object::ShouldThrowExceptions::Yes));
+          // e. Set k to k + 1.
+    }
+
+    return return_array;
+}
+
 // 1.2.2.1.4 %TypedArray%.prototype.toSorted ( comparefn ) https://tc39.es/proposal-change-array-by-copy/#sec-%typedarray%.prototype.toSorted
```

Comment:
Please also add `, ` between the signature and the URL here while you're at it :)